### DISCUSSION TITLE
How can I launch a shortcut I have on my desktop?

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [Ulauncher/Ulauncher#1260] [Q&A] [answered]
I have several shortcuts on my desktop, see example below. 

If I double click the shortcut, it opens up that program (or actually a particular file in a program, but that shouldn't matter).

If I try opening the file/program via ulauncher or rather the fuzzy search extension, it opens the symlink file as a text document. 

How can I launch the link directly as intended?

Thanks!

```
[Desktop Entry]
Name=Obsidian
Exec=test blabla"
Terminal=false
Type=Application
Icon=flower.png
StartupWMClass=
Comment=
MimeType=x-scheme-handler/obsidian;
Categories=Office;
Name[en_US]=Flower
```

**Comment by friday** [comment, marked as answer]
They should be in ~/.local/share/applications

See https://wiki.archlinux.org/title/desktop_entries for more info on how desktop entries work.

**Reply by ReaderGuy42** [reply]
Perfect, now they all appear! Thank you :)
